bin/prime.js

```diff
@@ -49,55 +46,11 @@ parser.command('erase')
   })
   .help('Erase pushed code from Tessel filesystem.');
 
-parser.command('root')
+parser.command('list')
   .callback(function(opts) {
-    root.sshRoot(opts);
+    controller.listTessels();
   })
-  .option('host', {
-    abbr: 'h',
-    help: 'The IP Address of the remote Tessel'
-  })
-  .option('keyPath', {
-    abbr: 'k',
-    help: 'The path to your SSH Key'
-  })
-  .option('keyPassphrase', {
-    abbr: ''
-  })
-  .help('Access the terminal of your remote Tessel');
-
-parser.command('wifi')
```

Comment:
Most of it will be added back in once we have USB comms but it's not that useful otherwise and I wanted to get this PR out quickly.